sdk/communication/AzureCommunicationChat/Source/ChatThreadClient.swift

```diff
@@ -318,21 +318,19 @@ public class ChatThreadClient {
 
     /// Updates a message.
     /// - Parameters:
+    ///    - message: The message id.
     ///    - content: The updated message content.
-    ///    - messageId: The message id.
     ///    - options: Update chat message options
     ///    - completionHandler: A completion handler that receives a status code on success.
     public func update(
-        content: String,
-        forMessage messageId: String,
+        message messageId: String,
+        content: UpdateChatMessageRequest,
         withOptions options: UpdateChatMessageOptions? = nil,
         completionHandler: @escaping HTTPResultHandler<Void>
     ) {
```

Comment:
If you are going to be exposed `UpdateChatMessageRequest` due to service changes, you should definitely _not_ expose this here like this, as the API Review Board will certainly not want you to introduce a breaking change immediately after GAing the SDK.